aptos-move/aptos-vm-types/src/abstract_write_op.rs

```diff
@@ -133,6 +154,7 @@ pub struct GroupWrite {
     pub(crate) inner_ops: BTreeMap<StructTag, (WriteOp, Option<Arc<MoveTypeLayout>>)>,
     /// Group size as used for gas charging, None if (metadata_)op is Deletion.
     pub(crate) maybe_group_op_size: Option<u64>,
+    pub(crate) prev_group_size: u64,
```

Comment:
I suppose this is 0 if group is being created?
I'm fine with it, just wondering whether we should have maybe_group_op_size and prev_group_size have a consistent convention. It's also fine to wait for later (can add a TODO)